app/views/guest_inbox.slim

```diff
@@ -0,0 +1,6 @@
+.container
```

Comment:
As mentioned, this can use the same view as a regular inbox

Reply:
Ok, done